work_with_shared_resources/using-shared-resource-csi-driver.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shared-resource-csi-driver"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: shared-resource-csi-driver
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource Container Storage Interface (CSI) Driver in any Kubernetes object that uses the volume type as `csi`. {builds-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {ocp-product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: 
+
+* `SharedSecret` custom resource for `Secret` objects
+* `SharedConfigMap` custom resource for `ConfigMap` objects
+
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-sharing-secrets-across-namespaces.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-sharing-configmaps-across-namespaces.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-additional-details-about-volumeattributes-on-shared-resource-pod-volumes.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc[leveloffset=+1]
+
+include::modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc[leveloffset=+1]
```

Comment:
@siamaksade :thinking: here when it comes to content strategy:

1. Should this content be moved to a separate article? We did this for BuildConfig when the driver was tech preview: https://docs.openshift.com/container-platform/4.17/cicd/builds/running-entitled-builds.html#builds-running-entitled-builds-with-sharedsecret-objects_running-entitled-builds
2. If yes - should it reside in the "Shared Resources" section, or the "Builds" section?

Reply:
Since this is specific to using the CSI driver, I think it's fine to keep it here. But I agree that we perhaps need a section under "Work with Builds" to detail how one can run entitled builds, using the CSI drive (link to this section) and also without it similar to the instructions for BuildConfigs.

Reply:
We'll add this to our stretch goals and cover it later